src/main/java/com/kirinpatel/sync/gui/VLCJMediaPlayer.java

```diff
@@ -54,10 +54,10 @@
         media  = new Media("");

```

Comment:
This would require a complete rework of the networking of sync. We should discuss if this is still something that we want to do.